wave_utils/include/wave/utils/math.hpp

```diff
@@ -34,6 +34,7 @@ void vec2mat(std::vector<double> x, int rows, int cols, MatX &y);
 void mat2vec(MatX A, std::vector<double> &x);
 double wrapTo180(double euler_angle);
 double wrapTo360(double euler_angle);
+Mat3 skew_symmetric_matrix(Vec3 a);
 
```

Comment:
I'd prefer naming that is easily understood first time around, without having to have familiarity with matlab's convention.  I find skewdec a bit unclear, where as wrapTo180 and wrapTo360 seem pretty clear already.